app/controllers/api/groups_controller.rb

```diff
@@ -37,6 +37,11 @@ def upload_photo
     respond_with_resource
   end
 
+  def export
+    service.export(group: load_and_authorize(:group), actor: current_user)
```

Comment:
Pedantic, but you should be able to do
```
service.export(group: load_resource, actor: current_user)
```
because it should be the service's job to authorize whether you can export or not.

Reply:
Kind of related. One thing that might be relevant is anonymous polls. 
A data export would allow people to see who voted what. 

If it were not for anonymous polls then I'd say that anyone who is a member should be able to export the group data.

If there are anonymous polls, I don't know if anyone should be able to download the data. It's a weird situation. What are your thoughts?